src/Lykke.Payments.Link4Pay.Domain/Lykke.Payments.Link4Pay.Domain.csproj

```diff
@@ -1,9 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
why downgrade?

Reply:
it's not easy to use .net core 3.1 with current infrastructure packages